### PR TITLE
Update front matter to include eithe es5id or es6id where missing. Closes gh-144

### PR DESCRIPTION
**Pull request by rwaldron**
- Corrects a handful of mis-labeled entries
- Adds missing `es5id`
- Adds missing `es6id`

This **does not** add the `es6id` for items that have `es5id`.

### CODE REVIEW

File: test/built-ins/Promise/prototype/race/S25.4.4.3_A2.2_T3.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 /*---
 info: > 
+es6id: S25.4.4.3_A2.2_T3
```
Comment (bterlson) [inline test/built-ins/Promise/prototype/race/S25.4.4.3_A2.2_T3.js]: Have to watch for the "info: >" form and put the es6id after it... looks like there are a few cases where es6id is inserted in a bad place.

Comment (rwaldron) [inline test/built-ins/Promise/prototype/race/S25.4.4.3_A2.2_T3.js]: Shit. I'll check them all.

**Comment by rwaldron** [comment]
@bterlson updated with fixes to those `info: >` misplacements

**Comment by bterlson** [comment]
Thanks! LGTM now.

**Pull request merged**